controller/volume_controller.go

```diff
@@ -50,7 +50,8 @@ var (
 )
 
 const (
-	CronJobBackoffLimit = 3
+	CronJobBackoffLimit               = 3
```

Comment:
I thinks it is already the default value for cron job set by K8S 
CronJobBackoffLimit               = 3
CronJobSuccessfulJobsHistoryLimit = 1
see https://v1-18.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#cronjob-v1beta1-batch

Reply:
@obeyler Thank you for looking into this!

In the link you gave, it said that the default value for `CronJobBackoffLimit = 6` , and `CronJobSuccessfulJobsHistoryLimit = 3`.

We want to reduce those number in Longhorn recurring job since they don't really need to keep that many.